### PR TITLE
VSTS - Add Reviewers & Assignees

### PR DESCRIPTION
**Pull request by jycouet**
Link to https://github.com/singapore/renovate/issues/1149

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -702,15 +702,15 @@ Obviously, you can't set repository or package file location with this method.
 </tr>
 <tr>
   <td>`assignees`</td>
-  <td>Assignees for Pull Request</td>
+  <td>Assignees for Pull Request (with email in vsts, user name in GitHub)</td>
```
Comment (rarkins) [inline docs/configuration.md]: This table should not be edited directly, as it gets regenerated regularly and hence would be overwritten later. Instead, please edit `lib/config/definitions.js` and run `npm run update-docs`

**Review by rarkins — CHANGES_REQUESTED** [review]

File: docs/configuration.md (hunk 1 above)
Comment (jycouet) [reply docs/configuration.md]: Hooo, sorry. I got it. I'll update this tonight.

Comment (jycouet) [reply docs/configuration.md]: Done 👍

**Comment by rarkins** [comment]
Thanks. Do we need to make any more changes to scripts to allow for windows?

**Comment by jycouet** [comment]
First it's a `.sh` script. So not native to windows. => Need to change this
Second I had problem with `\r` and `\n` => Need to investigate this

I would say that this is not link to this PR. You could create a ticket for that? With prio low?
And merge this one? :)

File: lib/config/definitions.js [hunk 2]
```diff
@@ -597,12 +597,14 @@ const options = [
   },
   {
     name: 'assignees',
-    description: 'Assignees for Pull Request',
+    description:
+      'Assignees for Pull Request (with email in vsts, user name in GitHub)',
```
Comment (rarkins) [inline lib/config/definitions.js]: This needs to say GitHub/GitLab

File: lib/config/definitions.js [hunk 3]
```diff
@@ -597,12 +597,14 @@ const options = [
...
     type: 'list',
   },
   {
     name: 'reviewers',
-    description: 'Requested reviewers for Pull Requests (GitHub only)',
+    description:
+      'Requested reviewers for Pull Requests (with email or user name in vsts, user name in GitHub)',
```
Comment (rarkins) [inline lib/config/definitions.js]: This also needs to say GitHub/GitLab

Comment (rarkins) [inline lib/config/definitions.js]: So assignees needs to be an email, but reviewers can be email or username?

Comment (jycouet) [reply lib/config/definitions.js]: Yes. Actually in VSTS you have a clear field for reviewers.
For assignees, I just create a comment with @<emailAddress> (this is the way to tag someone. Of course, when you do this in VSTS UI directly, you have autocompletion with the name)

I think it's the best we can do here

**Pull request merged**